### ISSUE TITLE
Crash on any model with 32k context

### ISSUE THREAD

**Issue by Jacoby1218** [LostRuins/koboldcpp#648]
crash is new with 1.56
```
Traceback (most recent call last):
  File "G:\koboldcpp\koboldcpp.py", line 2591, in <module>
    main(parser.parse_args(),start_server=True)
  File "G:\koboldcpp\koboldcpp.py", line 2437, in main
    loadok = load_model(modelname)
  File "G:\koboldcpp\koboldcpp.py", line 328, in load_model
    ret = handle.load_model(inputs)
OSError: exception: access violation reading 0x0000000000000000

**Comment by LostRuins** [comment]
Can check, does this still happen in 1.57?
If yes, please note the run parameters and the model used. Thanks.

**Comment by Jacoby1218** [comment]
Looks like this was fixed in 1.57,
